lms/djangoapps/mobile_api/users/tests.py

```diff
@@ -50,6 +69,9 @@ class TestUserEnrollmentApi(MobileAPITestCase, MobileAuthUserTestMixin, MobileCo
     REVERSE_INFO = {'name': 'courseenrollment-detail', 'params': ['username']}
     ALLOW_ACCESS_TO_UNRELEASED_COURSE = True
     ALLOW_ACCESS_TO_MILESTONE_COURSE = True
+    TOMORROW = datetime.datetime.now(pytz.UTC) + datetime.timedelta(days=1)
```

Comment:
As was pointed out to me by Nimisha when I did this same thing, this could cause Jenkins builds to fail in a really funny way near midnight :)